application/utilities/ttrequest.py

```diff
@@ -0,0 +1,94 @@
+import proto.traveltime.RequestsCommon_pb2 as RequestsCommon
```

Comment:
I'm not sure this isn't one of the primary classes that run the lambda function, I wouldn't have this in utilities, it's not for a standalone purpose like the other occasional use files in here.